app/views/medicaid/contact_social_security/edit.html.erb

```diff
@@ -7,14 +7,38 @@
 
   <div class="form-card__content">
     <%= form_for @step, as: :step, builder: MbFormBuilder, url: current_path, method: :put do |f| %>
-      <%= f.mb_input_field :last_four_ssn,
-        "Social Security Number (last 4 digits)",
-        type: :tel,
-        options: { maxlength: 4 },
-        notes: ["If you don’t know it you can skip this"],
-        append_html: "<p class='text--secure'><i class='illustration illustration--safety'></i>Social security numbers help ensure you receive the correct benefits. MDHHS maintains strict security guidelines to protect the identities of our residents.</p>" %>
+      <% if single_member_household? %>
+        <%= f.fields_for('members[]', current_application.primary_member) do |ff| %>
+          <%= ff.mb_input_field :last_four_ssn,
+            "Social Security Number (last 4 digits)",
+            type: :tel,
+            options: { maxlength: 4 },
+            notes: ["If you don’t know it you can skip this"],
+            append_html: "<p class='text--secure'><i class='illustration illustration--safety'></i>Social security numbers help ensure you receive the correct benefits. MDHHS maintains strict security guidelines to protect the identities of our residents.</p>" %>
 
-      <%= f.mb_date_select :birthday, "What is your birthday?", { options: { start_year: 1900, end_year: Time.now.year, default: Date.new(1990,1,15), order: [:month, :day, :year] } } %>
+          <%= ff.mb_date_select :birthday, "What is your birthday?", { options: { start_year: 1900, end_year: Time.now.year, default: Date.new(1990,1,15), order: [:month, :day, :year] } } %>
+        <% end %>
+      <% else %>
+        <% @step.members.select{|m| m.requesting_health_insurance? }.each do |member| %>
```

Comment:
small thing! we had talked about moving this filtering out of the view (maybe it belongs in the step or a shared step, since that's where it currently lives for providing info about your current health insurance)?

Reply:
good call! added a `requesting_health_insurance` scope on the member class at 98d764c0, which cleans this up a bit.